src/app/core/user-module/user.service.ts

```diff
@@ -95,6 +75,16 @@ export class UserService {
         );
   }
 
+  getUsersLearningObjects(username: string): Promise<LearningObject[]> {
+    return this.http
+      .get(USER_ROUTES.GET_USER(username), { withCredentials: true })
+      .pipe(catchError(this.handleError))
+      .toPromise()
+      .then((val: any) => {
+        return val.map((l) => new LearningObject(l));
```

Comment:
This will return a mostly empty LearningObject because it is trying to map a User to a LearningObject.